src/index.js

```diff
@@ -226,15 +226,18 @@ module.exports = (web3, options = {}) => {
           repo.methods.isValidBump(formatVersion(fromVersion), formatVersion(toVersion)).call()
         ))
     },
+    getKernel (app) {

```

Comment:
Seems unnecessary to introduce this into the public API. Could we move it up above the `return` so it's an internal method?